### DISCUSSION TITLE
Transaction Report better evaluation possibility

### DISCUSSION THREAD

**Discussion by renato-mmex** [moneymanagerex/moneymanagerex#5846] [Ideas]
As a "normal" user of MMEX you are not familiar with the creation of GRM.
For me it would be very helpful if there would be more evaluation possibilities in the transaction report.

Example 1:
I have a house, two cars, a health insurance and a legal protection insurance.
If I now want to know how much I have spent on all these insurances, I have to transfer the transactions into Excel and evaluate them here.

Example 2:
Because of the security I have divided our money on different accounts / banks.
If I now want to know how much money is in which bank, I again need Excel.
We have collective and separate accounts, again I need to resort to Excel.

**I would suggest that you could select multiple accounts, as well as multiple categories.**

What do you think about this ?

**Comment by vomikan** [comment]
It would be nice to modify the export of reports so that the report itself is uploaded to csv and xml. Make the remaining items inactive.

![image](https://user-images.githubusercontent.com/6836805/233045136-fd7991f0-e351-4ffa-9db1-51b431a7cdc5.png)
